modules/ebi-metagenomics/antismash/main.nf

```diff
@@ -0,0 +1,67 @@
+
+process ANTISMASH {
+    tag "$meta.id"
+    label 'process_medium'
+
+    // current antiSMASH bioconda container is buggy so we use our custom one (https://github.com/antismash/antismash/discussions/711)
+    container "microbiome-informatics/antismash:7.1.0.1_2"
+
+    input:
+    tuple val(meta), path(contigs)
+    tuple val(meta), path(genes)
+    tuple path(antismash_db), val(db_version)
+
+    output:
+    tuple val(meta), path("${meta.id}_results/${meta.id}.gbk")  , emit: gbk
+    tuple val(meta), path("${meta.id}_results/${meta.id}.json") , emit: json
+    tuple val(meta), path("${meta.id}_antismash.tar.gz")        , emit: results_tarball
+    path "versions.yml"                                         , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def is_compressed = antismash_db.name.endsWith(".tar.gz")
+    def antismashdb_name = antismash_db.name.replace(".tar.gz", "")
+
+    """
+    if [ "$is_compressed" == "true" ]; then
+        tar -xzvf $antismash_db
+    fi
+
+    antismash \\
+        -c ${task.cpus} \\
+        ${args} \\
+        --databases ${antismashdb_name} \\
+        --genefinding-gff3 ${genes} \\
+        --output-dir ${prefix}_results \\
+        ${contigs}
```

Comment:
We should supprot gbk inputs too. For example -> https://github.com/EBI-Metagenomics/mettannotator/blob/dev/modules/local/antismash.nf uses a gbk instead of a gff.

Or should we modify the mettannotator one instead? I'm not sure.

Reply:
I think more options are better - I'll rework the inputs so that it can be either GFF or GBK